go.mod

```diff
@@ -91,3 +91,5 @@ replace github.com/openstack-k8s-operators/glance-operator/api => ./api
 // mschuppert: map to latest commit from release-4.13 tag
 // must consistent within modules and service operators
 replace github.com/openshift/api => github.com/openshift/api v0.0.0-20230414143018-3367bc7e6ac7 //allow-merging
+
+replace github.com/openstack-k8s-operators/mariadb-operator/api => github.com/zzzeek/mariadb-operator/api v0.0.0-89bcc0f6f6941 //allow-merging
```

Comment:
this will also go away once we merge the mariadb-operator side of it (which we'll know is done when we're happy with how this looks)